### PR TITLE
Update kibana-deployment.yaml

### PR DESCRIPTION
**Pull request by zhangguanzhang**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:

/kind bug

**What this PR does / why we need it**:
the kibana's env vars name was error
**Which issue(s) this PR fixes**:
https://github.com/kubernetes/kubernetes/issues/80455

### CODE REVIEW

File: cluster/addons/fluentd-elasticsearch/kibana-deployment.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ spec:
           requests:
             cpu: 100m
         env:
-          - name: ELASTICSEARCH_URL
+          - name: ELASTICSEARCH_HOSTS
```
Comment (zhangguanzhang) [inline cluster/addons/fluentd-elasticsearch/kibana-deployment.yaml]: after 7.0, it should use this name

**Comment by zhangguanzhang** [comment]
@coffeepac Please help me to add the `/lgtm` label.

**Comment by coffeepac** [comment]
I'm a little confused as to how this works at all actually.  According to the docs this should cause kibana to fail to connect.  @zhangguanzhang have you tested this in a cluster before and after updating the yaml?

**Comment by zhangguanzhang** [comment]
@coffeepac you could have a look at this issue: https://github.com/kubernetes/kubernetes/issues/80455

**Comment by zhangguanzhang** [comment]
well....   It was submitted by someone else.
https://github.com/kubernetes/kubernetes/commit/19c0aa98e1e8eed91460be5827b239c25fe56567#diff-16d6720d733f4a8d336f41acfc43c857R31
```
lostick committed on 21 Jul
```

**Comment by coffeepac** [comment]
@zhangguanzhang ah, so it looks like this has already been fixed?  I vaguely remember this issue (I was moving across the country when this all landed).  

If this PR is a no-op can you close it?  should just be `/close`

**Comment by zhangguanzhang** [comment]
ah, I am submitting pr earlier than him.

**Comment by coffeepac** [comment]
@zhangguanzhang yeah, looks like you submitted a PR a day before him but his got merged.

thank you for the contribution and for being patient with me!  I'm no longer moving so if you do anything else in this area I'll be a lot faster.

**Pull request closed without merging**